app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -320,6 +326,29 @@ def test_check_experiment_pushes_experiment_before_ending_experiment(
         tasks.nimbus_check_kinto_push_queue()
         self.mock_end_task.assert_called_with(experiment_2.id)
 
+    def test_check_experiment_that_should_pause_does_pause(
+        self,
+    ):
+        experiment = NimbusExperimentFactory.create_with_status(
+            NimbusExperiment.Status.LIVE,
+            proposed_enrollment=10,
+            application=NimbusExperiment.Application.DESKTOP,
+        )
```

Comment:
And `is_paused=False` here too

Reply:
@jodyheavener bump?

Reply:
oh whoops

Reply:
what shouldn't that test be failing now?

Reply:
Oh I guess not if is_paused defaults to False

Reply:
@jaredlockhart updated

Reply:
Yeah it defaults to false but it's just a good practice to specify all the priors for a test to pass explicitly so if we ever change the factories the test won't suddenly fail.  Thnx!

Reply:
Agree, good call